content/en/docs/armory-agent/armory-agent-quick.md

```diff
@@ -1,25 +1,92 @@
 ---
-title: "Armory Agent for Kubernetes Quick Start Installation"
-linkTitle: "Quick Start"
+title: "Armory Agent for Kubernetes Quickstart Installation"
+linkTitle: "Quickstart"
 description: >
-  Learn how to install the Armory Agent in your Kubernetes and Spinnaker environments.
+  Learn how to install the Armory Agent in your Kubernetes and Armory Enterprise environments.
 weight: 2
 ---
 ![Proprietary](/images/proprietary.svg)
-> This guide is for experienced Kubernetes and Spinnaker<sup>TM</sup> users.
 
-## Compatibility matrix
+## {{% heading "prereq" %}}
+
+* This guide is for experienced Kubernetes and Armory Enterprise users.
+* You have read the Armory Agent [overview]({{< ref "armory-agent" >}}).
+* You have installed [MySQL storage for Clouddriver]({{< ref "clouddriver-sql-configure" >}}), which is required for the Agent.
```

Comment:
Note this will change to also include postgresql.
Another prerequisite is to have a redis instance, which is used by the Agent to coordinate between clouddriver replicas.